modules/react/button/lib/DeleteButton.tsx

```diff
@@ -15,50 +14,54 @@ import {systemIconStencil} from '../../icon';
+  base: {
+    // Base Styles
+    [buttonStencil.vars.background]: brand.error.base,
+    [buttonStencil.vars.borderRadius]: system.shape.round,
+    [buttonStencil.vars.label]: brand.error.accent,
     [systemIconStencil.vars.color]: brand.error.accent,
-    [buttonVars.focus.boxShadowInner]: base.frenchVanilla100,
-    [buttonVars.focus.boxShadowOuter]: brand.common.focusOutline,
-  },
-  '&:hover, &.hover': {
-    [buttonVars.hover.background]: brand.error.dark,
-    [buttonVars.hover.border]: 'transparent',
-    [buttonVars.hover.label]: brand.error.accent,
-    [systemIconStencil.vars.color]: brand.error.accent,
-  },
-  '&:active, &.active': {
-    [buttonVars.active.background]: brand.error.darkest,
-    [buttonVars.active.border]: 'transparent',
-    [buttonVars.active.label]: brand.error.accent,
-    [systemIconStencil.vars.color]: brand.error.accent,
-  },
-  '&:disabled, &.disabled': {
-    [buttonVars.disabled.background]: brand.error.base,
-    [buttonVars.disabled.label]: brand.error.accent,
-    [systemIconStencil.vars.color]: brand.error.accent,
-    [buttonVars.disabled.opacity]: '0.4',
+    // Focus Styles
+    '&:focus-visible, &.focus': {
+      [buttonStencil.vars.background]: brand.error.base,
+      [buttonStencil.vars.label]: brand.error.accent,
+      [systemIconStencil.vars.color]: brand.error.accent,
+      [buttonStencil.vars.boxShadowInner]: system.color.fg.inverse,
+      [buttonStencil.vars.boxShadowOuter]: brand.common.focusOutline,
+    },
+    // Hover Styles
+    '&:hover, &.hover': {
+      [buttonStencil.vars.background]: brand.error.dark,
+      [buttonStencil.vars.label]: brand.error.accent,
+      [systemIconStencil.vars.color]: brand.error.accent,
+    },
+    // Active Styles
+    '&:active, &.active': {
+      [buttonStencil.vars.background]: brand.error.darkest,
+      [buttonStencil.vars.label]: brand.error.accent,
+      [systemIconStencil.vars.color]: brand.error.accent,
+    },
+    // Disabled Styles
+    '&:disabled, &.disabled': {
+      [buttonStencil.vars.background]: brand.error.base,
+      [buttonStencil.vars.label]: brand.error.accent,
+      [systemIconStencil.vars.color]: brand.error.accent,
+      [buttonStencil.vars.opacity]: system.opacity.disabled,
+    },
   },
 });
 
 /**
  * Use sparingly for destructive actions that will result in data loss, can’t be undone, or will
  * have significant consequences. They commonly appear in confirmation dialogs as the final
- * confirmation before deleting.
+ * confirmation before being deleted.
  */
 export const DeleteButton = createComponent('button')({
   displayName: 'DeleteButton',
-  Component: ({children, size, ...elemProps}: DeleteButtonProps, ref, Element) => {
+  Component: ({children, ...elemProps}: DeleteButtonProps, ref, Element) => {
     return (
-      <Button as={Element} ref={ref} size={size} {...mergeStyles(elemProps, [deleteStyles])}>
+      <Button as={Element} ref={ref} {...handleCsProp(elemProps, deleteButtonStencil())}>
```

Comment:
Are we intentionally preventing the usage of `style` props? Doesn't `handleCsProp` not allow style props?

Reply:
This will be updated to use `mergeStyles`. 

We originally were going to go with `handleCsProp` but, I found an issue with other components using `TertiaryButton` and it was causing issues.